lib/genome/importers/tempus/tempus_tsv_importer.rb

```diff
@@ -0,0 +1,27 @@
+module Genome
+  module Importers
+    module Tempus
+      def self.source_info
+        {
+          base_url:           'https://www.tempus.com/',
+          site_url:           'https://www.tempus.com/',
+          citation:           'https://www.tempus.com/clinical-validation-of-the-tempus-xt-next-generation-targeted-oncology-sequencing-assay/',
```

Comment:
I would switch the base url and the citation url for to be more consistent with other importers.